### ISSUE TITLE
[millipede] fix the FIXME in python/artists.py (Trac #1249)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1243]
the FIXME should be fixed or converted into a TODO.

here is my suggestion on how to make the tables configurable by the user. i would add a string-setting, like so

(in __init__)
self.addSetting("Photon table", "$I3_DATA/photon-tables/splines/ems_mie_z20_a10")

and then parse and complete the string in create(...)

Migrated from https://code.icecube.wisc.edu/ticket/1249
```json
{
    "status": "closed", 
    "changetime": "2016-02-19T19:00:53", 
    "description": "the FIXME should be fixed or converted into a TODO.\n\nhere is my suggestion on how to make the tables configurable by the user. i would add a string-setting, like so\n\n(in __init__)\nself.addSetting(\"Photon table\", \"$I3_DATA/photon-tables/splines/ems_mie_z20_a10\")\n\nand then parse and complete the string in create(...)", 
    "reporter": "hdembinski", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1455908453690620", 
    "component": "combo reconstruction", 
    "summary": "[millipede] fix the FIXME in python/artists.py", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-08-20T17:07:42", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T20:58:52: 
* nega changed status from "new" to "assigned"
* nega changed owner from "jbraun@icecube.wisc.edu" to "jbraun"

**Comment by icecube-trac** [comment]
Trac update at 20160218T19:41:46: jbraun changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20160219T19:00:53: 
* jbraun changed status from "accepted" to "closed"
* jbraun changed resolution from "" to "fixed"
* jbraun commented:

> In changeset 142136:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="142136"
> Allow selection of photon tables from the GUI.  Closes #1243.
> ```
